Formula/jo.rb

```diff
@@ -21,6 +21,7 @@ class Jo < Formula
   def install
     system "autoreconf", "-i" if build.head?
 
+    ENV["bashcompdir"] = "#{prefix}/etc/bash_completion.d/"
```

Comment:
```suggestion
    ENV["bashcompdir"] = bash_completion unless OS.mac?
```

https://www.rubydoc.info/github/Homebrew/brew/Formula#bash_completion-instance_method

Reply:
Thanks for the hint! I'm going to commit the suggestion once we resolve the below issue.

Reply:
You can do it now. I think I'm going to merge this, since other maintainers have no clue either why this situation is happening and it won't harm to have this env var set.